en/console-and-shells/completion-shell.rst

```diff
@@ -94,20 +106,50 @@ The following bash example comes from the original author::
             opts=$(${cake} Completion subcommands $prev)
             COMPREPLY=( $(compgen -W "${opts}" -- ${cur}) )
             if [[ $COMPREPLY = "" ]] ; then
-                COMPREPLY=( $(compgen -df -- ${cur}) )
+                _filedir
```

Comment:
It's a function that comes from the bash-completion lib. 
https://code.google.com/p/bash-completion-lib/source/browse/trunk/include/_filedir?r=86

It was installed by default on my Mint install. It's probably a default on Ubuntu.
Which is why I specified that the file sample is to be used with a debian distribution and that the bash-completion library should be installed before using this.

Unfortunately, I do not know if it is bundled with other distributions or Mac OS (might be able to check that at work tomorrow though)